src/tests/ftest/util/soak_utils.py

```diff
@@ -355,6 +356,30 @@ def run_metrics_check(self, logging=True, prefix=None):
                     write_logfile(result["stdout"], log_name, destination)
 
 
+def display_job_failures(self):
```

Comment:
I'd recommend against using `self` when it's not actually a class method, since it might be confusing.